squad/core/comparison.py

```diff
@@ -108,37 +108,66 @@ def diff(self):
         return self.__diff__
 
     __regressions__ = None
+    __progressions__ = None
 
     @property
     def regressions(self):
-        if self.__regressions__ is not None:
+        return self.__regressions_progression__()
+
+    @property
+    def progressions(self):
+        return self.__regressions_progression__(False)
+
+    def __regressions_progression__(self, regression=True):
+        comparison_tuple = ('pass', 'fail')
+        if not regression:
+            comparison_tuple = ('fail', 'pass')
```

Comment:
maybe instead passing a boolean and doing this test, you could pass in the comparison you want explicitly?

Reply:
Hmm, might be a good idea. We can do n/a->pass (or fail) transitions as well. Not sure if they're useful but I heard a requirement for that some time ago.